projects/js-packages/publicize-components/src/components/add-connection-modal/style.module.scss

```diff
@@ -1,12 +1,5 @@
 .modal.modal {
 	width: 65rem;
```

Comment:
Maybe we could add this same width to the confirmation-from component as well?

Reply:
It's the same.

Reply:
Oh is it? On the demo video it's not, maybe it changes since?

Reply:
Demo is right. Do you mean height or width?

Reply:
I meant width, but yeah it was just my eyes then! 😃

Reply:
Confirmation uses the same modal. So, it's not any different modal.